ci_framework/roles/build_openstack_packages/tasks/create_repo.yml

```diff
@@ -33,6 +33,17 @@
     cmd: createrepo gating_repo
```

Comment:
I was looking at those lines, doesn't look like we do in quickstack.
if rpms are copied to `{{ cifmw_bop_gating_repo }}`, the createrepo cmd should be:
`createrepo {{ cifmw_bop_gating_repo }}` or `createrepo .` since you already do a `chdir` here.

Did I get it wrong?

Reply:
In quickstart we run `createrepo` here https://opendev.org/openstack/tripleo-quickstart-extras/src/branch/master/roles/build-test-packages/tasks/main.yml#L296 and we generate the repo files here: https://opendev.org/openstack/tripleo-quickstart-extras/src/branch/master/roles/install-built-repo/templates/install-built-repo.sh.j2#L8

Reply:
Yes, and as you can see, rpms are copied to gating_repo dir[1]. That is the point that I was trying to bring here,

[1] https://opendev.org/openstack/tripleo-quickstart-extras/src/commit/34cb52dfd23cc3314cd121aef8e8c56a99b314bb/roles/build-test-packages/tasks/main.yml#L292

Reply:
ok, just a heads-up if something is not working as expected, we can revisit these lines. But they are actually merged here, so nothing to do in this PR.

Reply:
cmd `createrepo .` should resolve that for us I think. Thanks